### PR TITLE
redirect /nix-dev to new location

### PR DESCRIPTION
**Pull request by garbas**

### CODE REVIEW

File: netlify.toml [hunk 1]
```diff
@@ -10,6 +10,12 @@
   status = 302
   force = true
 
+[[redirects]]
+  from = "/nix-dev/*"
+  to = "https://releases.nixos.org/nix-dev/:splat"
```
Comment (garbas) [inline netlify.toml:15]: @edolstra In this PR I'm assuming that I will upload nix-dev to nix-releases bucket. Would it be better to create new bucket for this eg. old.nixos.org or archive.nixos.org?

Comment (samueldr) [reply netlify.toml:15]: If it is in another bucket, I still strongly suggest suffixing with `/nix-dev/:splat` so we can archive more things as required.

Comment (edolstra) [reply netlify.toml:15]: Yeah let's use releases.nixos.org. Otherwise we'll have to set up another bucket + Cloudfront distribution + DNS.

Comment (garbas) [reply netlify.toml:15]: consider it done

**Pull request merged**